app/ante/ante.go

```diff
@@ -70,7 +70,6 @@ func NewAnteHandler(
 						NewEthNonceVerificationDecorator(ak),
 						NewEthGasConsumeDecorator(ak, bankKeeper, evmKeeper),
 						NewCanTransferDecorator(evmKeeper),
-						NewAccessListDecorator(evmKeeper),
```

Comment:
what is the issue with this decorator? the logic is pretty much the same one

Reply:
The issue is ante handler is not executed for `eth_call`.

Reply:
ok that makes sense. Why aren't we using the `Simulate` gRPC method again? then we could easily solve this issue from arising in the future if we add more logic to the ante handler

Reply:
`Simulate` don't support execute on historical block height, and cosmos-sdk team seems reluctant to support that https://github.com/cosmos/cosmos-sdk/issues/9636#issuecomment-874687473.

Reply:
Also to support `eth_call` in ante, we need to further adapt the ante handler logic, since in `eth_call` the `MsgEthereumTx` is not signed, and `from` could be empty.